content/release/ReleaseNotes372.md

```diff
@@ -0,0 +1,113 @@
+<!--
+title: "Contrast 3.7.2 - March 2020"
+description: "Contrast 3.7.2 March 2020"
+tags: "3.7.2 March Monthly Release News"
+-->
+
+## On-premises release date:
+March 31, 2020
+
+### New features and improvements
+
+**Python for Assess**
+This feature is now generally available for all Assess users to instrument web applications written in Python to find custom code vulnerabilities. Assess capabilities include support for Django, Flask, and Pyramid along with route coverage and route-based auto-verification for each framework.
+
+### Important note
+With the 3.7.1 release, which upgraded Tomcat to version 9, users were experiencing a broken AJP (Apache JServ Protocol) connection. This required an AJP secret, or that the secret be disabled. [Configuration options](admin-systemsettings.html#https) have been added and are configurable through the *server.properties* file.
+
+
+## Agent Updates
+
+### Java Agent Summary
+
+**Agent versions released this month:** 3.7.2.14224, 3.7.2.14458
+
+**Language versions supported:** Java 1.6 - 1.8 and 11
+
+#### Bug Fixes:
+* In some cases, when users sanitized attacker-controlled data encoders, Contrast Assess considered the encoder input to also be safe in other contexts. This could have led to false negatives in our data flow analysis. We fixed this so that only the data returned by an encoder (and never the input) would be considered safe in a data flow analysis.
```

Comment:
The phrasing of the first sentence is really hard to follow. Would the following be equivalent in meaning? 

In some cases, Contrast considered the input to user-specified sanitizers to also be safe in other contexts.

Reply:
When you change "users sanitized" to "user-specified sanitizers", you change the semantics of the sentence in such a way that it is no longer accurate. I think it works if we just remove "user-specified"

> In some cases, Contrast considered the input to sanitizers to also be safe in other contexts.

Reply:
Keeping Assess:
In some cases, Contrast Assess considered the input to sanitizers to also be safe in other contexts.